general/README.md

```diff
@@ -304,6 +305,65 @@ _handleEvent = function(e) {
 
 **[⬆ back to top](#table-of-contents)**
 
+## Iterators
+
+It is best not to use iterators. It is preferred to leverage JavaScript's higher-order functions when looping.
```

Comment:
this sentence might be confusing, all the mentioned above are iterators (High order iterators) 

we particularly try to avoid iterations that don't have any output as a result of the iteration, hence needing an external element to store any kind of operation (or transform an pre existent element as a result) to produce a valuable output. this is to say: `forEach` and `for-loops` and `while`